config/datatypes_conf.xml.sample

```diff
@@ -1 +0,0 @@
-../lib/galaxy/config/sample/datatypes_conf.xml.sample
```

Comment:
this looks wrong ... the link is the new way ... the real file should be removed.

Reply:
Ah, my mistake.